GoogleSignIn/Sources/GIDSignIn.m

```diff
@@ -180,6 +180,10 @@ - (void)restorePreviousSignInWithCallback:(nullable GIDSignInCallback)callback {
   [self signInWithOptions:[GIDSignInInternalOptions silentOptionsWithCallback:callback]];
 }
 
+- (void)restorePreviousSignInNoRefresh {
+  [self signInWithOptions:[GIDSignInInternalOptions silentOptionsNoRefresh]];
```

Comment:
I see a number of issues with the `signInWithOptions` approach.  Consider simply calling `restoreCurrentUserWithAuthState` here.  `signInWithOptions` is geared for an async flow.

Reply:
Done.